src/locales/en.json

```diff
@@ -178,7 +178,8 @@
   "doctypes": {
     "com": {
       "tricount.expenses": "Expenses Tricount",
-      "splitwise.expenses": "Expenses Splitwise"
+      "splitwise.expenses": "Expenses Splitwise",
+      "deezer.playlists": "Deezer playlists"
```

Comment:
Why playlists specific to deezer instead of a standard playlist? Is it so specific that we have to have an explicit doctype for it?

Reply:
The standard playlist is not defined yet, and we want the connector out. We will standardize a playlist doctype later

Reply:
What is the format of this doctype? Why can't we take a time to standardise that? At least a first version even if we have to iterate on it later.

Reply:
This was decided with most concerned people.

For now every konnector creates its doctype. Once we have 2-3 konnector importing the same kind of data and 1 application using it, we will try to figure out a common `io.cozy.xxxx` doctype and use a service to convert them before migrating the konnectors.